lib/jekyll-github-metadata/repository.rb

```diff
@@ -64,6 +64,10 @@ def releases_url
         "#{repository_url}/releases"
       end
 
+      def latest_release_url
+        "#{releases_url}/latest"
```

Comment:
Do you mean remove `latest_release_url` because as i just saw `url` is already included in the payload from github so the user could write `{{ site.github.latest_release.url }}` ? Or do you mean calling (if possible) `latest_release.url` instead of `"#{releases_url}/latest"` on this line?